src/engine/ruleset/decoders/sonicwall/firewall-general.yml

```diff
@@ -0,0 +1,77 @@
+---
+name: decoder/sonicwall-firewall-general/0
```

Comment:
If this is the general case, maybe it should be named just `sonicwall-firewall`, what do you think?